src/Arcus.WebApi.Tests.Integration/Logging/RequestTrackingMiddlewareTests.cs

```diff
@@ -1069,9 +1065,9 @@ await using (var server = await TestApiServer.StartNewAsync(options, _logger))
                 {
                     // Assert
                     Assert.Equal(HttpStatusCode.OK, response.StatusCode);
-                    IDictionary<string, string> requestLogProperties = GetRequestLogProperties(spySink);
+                    var requestLogProperties = spySink.GetRequestLogProperties();
```

Comment:
We should probably use the type itself here, as we describe in our code style rules, as you can't know the type beforehand here.